### ISSUE TITLE
Log output understanding question

### ISSUE THREAD

**Issue by mobilutz** [KnapsackPro/knapsack_pro-ruby#159]
I am seeing output from this line in our CI logs, and want to know what it means:
```
If you need to reproduce a particular subset of tests fetched from API queue then above after each request to Knapsack Pro API you will find example rspec command.
```

It comes form here: https://github.com/KnapsackPro/knapsack_pro-ruby/blob/3aa419270b40d7c6b6f6503ffa980d61c802bb7a/lib/knapsack_pro/runners/queue/rspec_runner.rb#L115

There seems to be something wrong with this sentence, especially here:
`... fetched from API queue then above after each request ...`

I would like to support, but I don't know what exactly this sentence means.

**Comment by shadre** [comment]
@mobilutz thanks for raising the issue. :+1: The bit you quoted is referring to other messages that come from [L113 in the same file](https://github.com/KnapsackPro/knapsack_pro-ruby/blob/3aa419270b40d7c6b6f6503ffa980d61c802bb7a/lib/knapsack_pro/runners/queue/rspec_runner.rb#L113). Basically, the command displayed below the message you are quoting would run all the test files assigned to a particular CI node, while the previous commands run just subsets of those (as retrieved from the API queue on each request).

We agree the turn of phrase is not particularly legible there. I'm going to prepare a change for this shortly. :-)

**Comment by mobilutz** [comment]
> The bit you quoted is referring to other messages that come from [L113 in the same file](https://github.com/KnapsackPro/knapsack_pro-ruby/blob/3aa419270b40d7c6b6f6503ffa980d61c802bb7a/lib/knapsack_pro/runners/queue/rspec_runner.rb#L113).

Doesn't the case statement prevent the output of L113 and L115 at the same time?

But 👍🏻 and thanks on a wording change to make it more understandable!

**Comment by shadre** [comment]
> Doesn't the case statement prevent the output of L113 and L115 at the same time?

Yes, it does. However, we expect this code to usually be run multiple times in the Queue Mode, resulting in multiple outputs (with the one from L115 coming last) - hence the reference to the commands "above". :-)

**Comment by ArturT** [comment]
@mobilutz We've released a new knapsack_pro version 3.1.1 with [updated messages](https://github.com/KnapsackPro/knapsack_pro-ruby/pull/160).
